src/dark-mode-toggle.mjs

```diff
@@ -299,7 +297,7 @@ export class DarkModeToggle extends HTMLElement {
         doc.querySelectorAll(`link[rel="stylesheet"][media="${MQ_DARK}"]`);
     this._lightCSS = document.querySelectorAll(MQ_LIGHT.map((mqLight) => {
       return `link[rel="stylesheet"][media*="${mqLight}"]`;
-    }).join(', '));
+    }).join());
```

Comment:
The space after the comma is not needed, so it could be `','`. It turns out that's the implicit default for `join`, so we don't even need to pass a string.

Reply:
Oh, [TIL](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join#Parameters). I never questioned the need for a separator. Nice.